src/utilities/io.jl

```diff
@@ -200,7 +200,7 @@ function Base.vcat(c1::Chain, args::Chain...)
   Chain(0, value2)
 end
 
-save!(c::Chain, spl::Sampler, model::Function, vi::VarInfo) = begin
+save!(c::Chain, spl::Sampler, model::Function, vi) = begin
```

Comment:
The reason is that `VarInfo` is defined in a file that's `include`d after the `utilities` files. This would error after moving the `io.jl` to the `utilities` folder, unless I include this file before and the rest of the utilities files after, which looked weird. Removing the unnecessary `VarInfo` annotation was more elegant.